fuse/requirements.txt

```diff
@@ -8,8 +8,10 @@ scipy>=1.5.4
 matplotlib>=3.3.3
 scikit-learn>=0.23.2
 termcolor>=1.1.0
+pycocotools>=2.0.1
 torch>=1.5.0
 torchvision>=0.8.1,<0.14.0
+pytorch_lightning
```

Comment:
>=1.6

Reply:
@mosheraboh 

Thanks, nice catch.
Just to make sure: it should be `>=1.7.6` (in order to support custom batch_sampler in DDP mode